content/en/agent/logs/log-transport.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Agent Transport for logs
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs sent to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-from-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data from your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+
+## Default agent behavior
```

Comment:
I think we should drop the Proxy note here (enforce TCP).
In addition, should we mention here that any config already set will be respected (when use_http=true/false is explicitly set)

Reply:
This is part of the enforce transport section so I believe it is fine to leave it that way.

I would not mix proxy information right at the beginning. Let's keep the proxy part separate.

Reply:
by proxy note you mean "HTTPS Proxy configuration" section or the "Sock5" note?

Reply:
If it's socks5, it's done,!

Reply:
lol just saw Nil's comment, moving back Socks5 ... :D